StPicoDstMaker/StPicoEvent.h

```diff
@@ -126,6 +140,14 @@ class StPicoEvent : public TObject {
   UShort_t       mRefMult2PosEast; // TPC refMult2 pos (-1<eta<-0.5)
   UShort_t       mRefMult2NegWest; // TPC refMult2 neg (0.5<eta<1.0)
   UShort_t       mRefMult2PosWest; // TPC refMult2 pos (0.5<eta<1.0)
+  UShort_t       mRefMult3NegEast; // TPC refMult3 neg (-1<eta<0.)
+  UShort_t       mRefMult3PosEast; // TPC refMult3 pos (-1<eta<0.)
+  UShort_t       mRefMult3NegWest; // TPC refMult3 neg (0.<eta<1.0)
+  UShort_t       mRefMult3PosWest; // TPC refMult3 pos (0.<eta<1.0)
+  UShort_t       mRefMult4NegEast; // TPC refMult4 neg (-1<eta<0.)
+  UShort_t       mRefMult4PosEast; // TPC refMult4 pos (-1<eta<0.)
+  UShort_t       mRefMult4NegWest; // TPC refMult4 neg (0.<eta<1.0)
+  UShort_t       mRefMult4PosWest; // TPC refMult4 pos (0.<eta<1.0)
```

Comment:
@xuji19891014 
Either remove the comment or make it clearer. As it stands, the comments don't distinguish between refMult3 and refMult4.